download_PL_data.py

```diff
@@ -0,0 +1,42 @@
+import requests
+import pandas as pd
+import os
+
+initial_year = 8485
+year = initial_year
+
+#Download and save data previous to season 2018/2019
+
+while year != 1819:
+    year = year + 101
+    if (year==10000):
+        year = 1
```

Comment:
All this is quite cryptic, what about adding some comments or using more explicit variable names?

Reply:
The comment regarding more explicit variable names also applies to the following variable "a"